structs.go

```diff
@@ -1054,42 +1054,6 @@ type APIErrorMessage struct {
 	Message string `json:"message"`
 }
 
-// Webhook stores the data for a webhook.
-type Webhook struct {
```

Comment:
This Pull Request about `slash command`, why you change Webhook ?

Reply:
Interaction Responses and Follow up messages use web hooks

See https://discord.com/developers/docs/interactions/slash-commands#edit-original-interaction-response

Reply:
> This Pull Request about `slash command`, why you change Webhook ?

I thought that better will be to separate everything with webhooks into dedicated file.